### PR TITLE
Phoebus removed

### PR DESCRIPTION
**Pull request by rbatistacardoso**
Parts of code relating to foebus installation have been removed. A "playbook-desktiops.yml" manual was tested after the change to see if nothing was confirmed.

### CODE REVIEW

File: inventories/sirius/group_vars/molecule_overrides.yml [hunk 1]
```diff
@@ -30,10 +30,4 @@ csstudio_xulrunner: "https://artifactory.esss.lu.se/artifactory/swi-pkg/xulrunne
...
 
 epics7_base_url: "https://epics.anl.gov/download/base"
-epics7_base_archive: "{{ epics7_base_url }}/base-{{ epics7_base_version }}.tar.gz"
-
-# === Phoebus ===
-
-phoebus_repository: nightly
-phoebus_base_url: "https://controlssoftware.sns.ornl.gov/css_phoebus"
-phoebus_archive: "{{ phoebus_base_url }}/{{ phoebus_repository }}/phoebus-linux.zip"
+epics7_base_archive: "{{ epics7_base_url }}/base-{{ epics7_base_version }}.tar.gz"
```
Comment (xresende) [inline inventories/sirius/group_vars/molecule_overrides.yml:33]: no newline at end of file

File: playbooks/playbook-desktops.yml [hunk 2]
```diff
@@ -55,4 +53,4 @@
         name: "lnls-ans-role-glusterfs"
       when: "'control_room_all' in group_names"
     - include_role:
-        name: "lnls-ans-role-nfsclient"
+        name: "lnls-ans-role-nfsclient"
```
Comment (xresende) [inline playbooks/playbook-desktops.yml:56]: no newline at end of file

File: inventories/sirius/group_vars/molecule_overrides.yml (hunk 1 above)
Comment (rbatistacardoso) [reply inventories/sirius/group_vars/molecule_overrides.yml]: Newline was added.

File: playbooks/playbook-desktops.yml (hunk 2 above)
Comment (rbatistacardoso) [reply playbooks/playbook-desktops.yml]: Newline was added.

**Review by lerwys — COMMENTED** [review]
I think this is fine. Some minor comments, not really related to this PR:

* Your git commit user is not linked to your account. Probably because your email is not the same as the one registered on github. Maybe this is already fixed.
* We generally use commit messages in english, just because we might end up collaborating with other lab or foreign person
* Commit messages should not exceed 80 characters. If needed you can write a more detailed message after the first line.

Overall it seems good!

Thanks!

**Comment by rbatistacardoso** [comment]
OK. Thanks for the clarifications, I will pay attention to them in the next commit. Thank you!

File: playbooks/playbook-desktops.yml (hunk 2 above)
Comment (xresende) [reply playbooks/playbook-desktops.yml]: @rbatistacardoso , my 'changed files' still shows a missing newline at end of file. I took the liberty to push commit to fix it, ok?

**Review by xresende — APPROVED** [review]

**Pull request merged**